internal/exec/resolvable/resolvable.go

```diff
@@ -158,6 +226,44 @@ func ApplyResolver(s *types.Schema, resolver interface{}, dirVisitors []directiv
 	}, nil
 }
 
+func buildDirectivePackers(s *types.Schema, visitors map[string]directives.Directive) (map[string]*packer.StructPacker, error) {
+	// Directive packers need to use a dedicated builder which is ready ('finish()' called) while
+	// schema fields (and their argument packers) are still being built
+	builder := packer.NewBuilder()
+
+	packers := map[string]*packer.StructPacker{}
+	for _, d := range s.Directives {
+		n := d.Name
+
+		v, ok := visitors[n]
+		if !ok {
+			// Directives which need visitors have already been checked
+			// Anything without a visitor now is an in-built directive without a packer.
+			continue
+		}
+
+		if _, ok := v.(directives.ResolverInterceptor); !ok {
```

Comment:
Apparently, when you start a draft PR, once you click "Ready for review" it bypasses checks by SemaphoreCI 🤦‍♂️ 
This marks master as broken because of [the same issue last time](https://graph-gophers.semaphoreci.com/jobs/83bb4db9-e210-49d3-a1e4-df0a7e31f4ef). I'll see how I can fix it.